finagle-http/src/test/scala/com/twitter/finagle/http/AbstractEndToEndTest.scala

```diff
@@ -916,4 +918,138 @@ abstract class AbstractEndToEndTest extends FunSuite
     await(server.close())
     await(client.close())
   }
+
+  testIfImplemented(NoBodyMessage)(
```

Comment:
I have a preference for small tests, do you mind structuring this differently so that the `test` clause is *inside* of the loop?  We can get around the duplicate names by string interpolating into the name of the test.